configuration/README.md

```diff
@@ -59,3 +59,9 @@ To support execution in a simulated environment, V2X-Hub is in the process of in
 * **V2XHUB_IP** – Environment variable for storing IP address of V2X Hub.
 * **INFRASTRUCTURE_ID** – Environment variable for storing infrastructure id of V2X Hub.
 * **SENSOR_JSON_FILE_PATH** – Environment variable for storing path to sensor configuration file. This is an optional simulation environment variable that allows for setting up simulated sensor for a V2X-Hub instance. Example file can be found in the **CDASimAdapterPlugin** tests [here](../src/v2i-hub/CDASimAdapter/test/sensors.json).
+
+### Open V2X-Hub in Google Chrome
+ To open V2X-Hub in Google Chrome, run this command:
```

Comment:
This  is the wrong command since we use chromium below. We can offer this as an option of the user has google chrome instead of chromium. A note explaining why we do this may be useful. Something link

> [!NOTE]  
> V2X-Hub Server uses a secure WebSocket connection to communicate with the browser. V2X-Hub Server uses a self-signed certificate instead of one signed by a trust certificate authority (CA) for this secure connection, which most browsers will not trust until explicitly told to do so by navigating the the WebSocket URL (https://<v2xhub_ip>:19760) and accepting the risk . The `--ignore-certificate-errors` option instructs the browser to ignore certificate errors and warnings which removes manually effort of accepting this risk on V2X-Hub deployments. **Using the browser with this option enabled to access the internet or any other application is not advised as it is a security risk**